.github/workflows/release.yml

```diff
@@ -89,7 +89,9 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: dist
-          path: viamrobotics-sdk-*.tgz
+          path: |
+            viamrobotics-sdk-*.tgz
+            docs/dist
```

Comment:
trying to remember - do we need to upload artifacts for any reason other than to break up the prepare and release jobs? if not maybe it's easier to combine them and remove all these artifact steps...

Reply:
I believe so, I think it's still worth separating the steps out tho